docker/Dockerfile

```
@@ -1,13 +1,23 @@
-FROM alpine:3.5
+# build the server binary
+FROM golang:1.11.0 AS builder
```

```suggestion
 FROM golang:1.11 AS builder
```
The current stable release is `1.11.4`. You can do `1.11` to catch all the patches on the 1.11 version.

---

gotchu